inference-engine/src/cldnn_engine/cldnn_config.h

```diff
@@ -59,6 +60,7 @@ struct Config {
     std::string graph_dumps_dir;
     std::string sources_dumps_dir;
     std::string device_id;
+    std::string kernels_cache_dir;
```

Comment:
I think we need to take into account that path specified by a user can contain unicode symbols.

Reply:
added unicode support